comp/metadata/host/hostimpl/utils/host_windows.go

```diff
@@ -38,29 +39,12 @@ var (
 	procGetTickCount64 = modkernel.NewProc("GetTickCount64")
 )
 
-// InfoStat describes the host status.  This is not in the psutil but it useful.
-type InfoStat struct {
-	Hostname             string `json:"hostname"`
-	Uptime               uint64 `json:"uptime"`
-	BootTime             uint64 `json:"bootTime"`
-	Procs                uint64 `json:"procs"`           // number of processes
-	OS                   string `json:"os"`              // ex: freebsd, linux
-	Platform             string `json:"platform"`        // ex: ubuntu, linuxmint
-	PlatformFamily       string `json:"platformFamily"`  // ex: debian, rhel
-	PlatformVersion      string `json:"platformVersion"` // version of the complete OS
-	KernelVersion        string `json:"kernelVersion"`   // version of the OS kernel (if available)
-	KernelArch           string `json:"kernelArch"`
-	VirtualizationSystem string `json:"virtualizationSystem"`
-	VirtualizationRole   string `json:"virtualizationRole"` // guest or host
-	HostID               string `json:"hostid"`             // ex: uuid
-}
-
```

Comment:
@iglendd is this change intentional? 

I'm going to flag to @hush-hush who wrote the host implementation to make sure we are ok with this change

Reply:
Looking through the history this was linked to no longer using WMI and removing usage of gopsutil from windows (#1725).

This code has been moved a number of times, we should definitely add a comment to explain this.